src/openzaak/components/catalogi/openapi.yaml

```diff
@@ -11361,105 +11214,6 @@ components:
       - informatieobjectcategorie
       - omschrijving
       - vertrouwelijkheidaanduiding
-    Notificatie:
```

Comment:
Makes me wonder why this and the callbacks above were here in the first place :sweat_smile:

Reply:
They appeared in the OAS after moving to drf_spectacular. We hid them in `drf_yasg`, but I forgot to hide them also in the `drf-spectacular`